mobify/astro-scaffold
# Hyb 500 update scaffold basic ecommerce functionality

Add ecommerce functionality to the scaffold - tabs, header bar and cart badge icon

Status: **Opened for visibility** - changes need to be made to Android in order for this to function on both platforms  
Reviewers: @jansepar @jeremywiebe @jvoll @kschmidtdev 
JIRA: https://mobify.atlassian.net/browse/HYB-500
Linked PRs: https://github.com/mobify/generator-astro/pull/7
## Changes
- Add a tabBarController, tabController and tabHeaderController
- Update astro
- Add a custom counter badge plugin
## How to test-drive this PR
- Run scaffold app (ios) and ensure tabs and header are present
- Tabs work as expected
## Research resources
- none
## TODOS:
- [ ] Deploy the updated docs
- [ ] Update the [API feature parity spreadsheet](https://docs.google.com/spreadsheets/d/1bQTjiInDM4A1glbLo_CI3FYx6iW3sfL0ykcz8tVRSkw/edit#gid=0)
- [ ] Update the scaffold if necessary.
